.travis.yml

```
@@ -274,6 +274,15 @@ matrix:
         - ! git grep -q --ignore-case "gnu general public";
         - ! git grep -q --ignore-case "gnu library general public";
 
+    - env:
+        - NAME=include_check
+      script:
+        - echo 'Checking that there are no '#include "mbed.h"' in code where it should not be'
+        - >-
```

My mistake on the suggestion. I think the following should fix the excludes:
```suggestion
        - |
```

Reference: https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines#21699210